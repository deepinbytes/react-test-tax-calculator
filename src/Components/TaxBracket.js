import React, { Component } from "react";
import Typography from '@material-ui/core/Typography';



class TaxBracket extends Component {

    render() {
        return (
            <div>
                <Typography variant="h4">
                    Income Tax Brackets
                </Typography>
                <Typography variant="subtitle1">Indonesian Tax Rates</Typography>
                <ol>
                    <li><Typography variant="body1">Annual Income from 0 to 50.000.000 IDR - tax rate is 5 %.</Typography></li>
                    <li><Typography variant="body1">Annual Income from 50.000.000 to 250.000.000 IDR - tax rate is 15 %.</Typography></li>
                    <li><Typography variant="body1">Annual Income from 250.000.000 to 500.000.000 IDR - tax rate is 25 %.</Typography></li>
                    <li><Typography variant="body1">Annual Income above 500.000.000 IDR - tax rate is 30 %.</Typography></li>
                </ol>
            </div>
        )
    }
}

export default TaxBracket;