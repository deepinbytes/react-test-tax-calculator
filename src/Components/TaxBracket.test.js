import React from 'react';
import renderer from 'react-test-renderer';

import TaxBracket from './TaxBracket';

it('taxbracket renders correctly', () => {
    const tree = renderer.create(<TaxBracket />).toJSON();
    expect(tree).toMatchSnapshot();
});