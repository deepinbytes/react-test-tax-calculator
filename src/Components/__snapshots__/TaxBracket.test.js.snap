// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`taxbracket renders correctly 1`] = `
<div>
  <h4
    className="MuiTypography-root MuiTypography-h4"
  >
    Income Tax Brackets
  </h4>
  <h6
    className="MuiTypography-root MuiTypography-subtitle1"
  >
    Indonesian Tax Rates
  </h6>
  <ol>
    <li>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Annual Income from 0 to 50.000.000 IDR - tax rate is 5 %.
      </p>
    </li>
    <li>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Annual Income from 50.000.000 to 250.000.000 IDR - tax rate is 15 %.
      </p>
    </li>
    <li>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Annual Income from 250.000.000 to 500.000.000 IDR - tax rate is 25 %.
      </p>
    </li>
    <li>
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Annual Income above 500.000.000 IDR - tax rate is 30 %.
      </p>
    </li>
  </ol>
</div>
`;
